### PR TITLE
Update `getJigsawJDK.sh` to the new Jigsaw downlaod page

### PR DESCRIPTION
**Pull request by lasombra**
This should fix issue #65

### CODE REVIEW

**Comment by neomatrix369** [comment]
@lasombra if you want to resolve #66 you ccan already do it

you will have to save this script file with a different name, which will make two script files available in the system:
```
getJDK.sh
getJigsawJDK.sh
```

Are you able to get my drift?

Do you want to create a new PR with those changes?

File: getJigsawJDK.sh [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 
 JDK_DESTINATION=$(echo ```pwd```)
 JDK_FOLDER_NAME="jdk-9"
 JDK_HOME_OS_SPECIFIC="$JDK_DESTINATION/$JDK_FOLDER_NAME"
 
-JDK_DOWNLOAD_HOME_PAGE="http://jdk.java.net/9/"
-BUILD_NUMBER=$(curl $JDK_DOWNLOAD_HOME_PAGE/ | grep "Most recent build" | awk '{print $4}' |  tr -d "</h2>" | grep -oP "jdk\-9\+\K\w+")
-JDK_DOWNLOAD_BASE_URL="http://download.java.net/"
+JDK_DOWNLOAD_HOME_PAGE="http://jdk.java.net/jigsaw/"
+BUILD_NUMBER=$(curl $JDK_DOWNLOAD_HOME_PAGE/ | grep "Most recent build" | awk '{print $4}' |  tr -d "</h2>" | perl -nle 'print$& if m{jdk\-9\+\K\w+}')
```
Comment (neomatrix369) [inline getJigsawJDK.sh]: I will have to test this change on my Ubuntu box before I merge it

Comment (lasombra) [inline getJigsawJDK.sh]: Cool. I tested on my RHEL/CentOS 6 box and it worked fine.

**Comment by lasombra** [comment]
Yeah. Let me work on it.

**Comment by neomatrix369** [comment]
Works fine on my ubuntu box as well - would be great to have some others test it out. But safe to merge now.

**Pull request merged**